Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.6
+
+COPY . /usr/src/app/
```

Comment:
It is a good practice to copy project files as late in the process as possible. Docker is using caches for builds, but they are based on the previous states, so things that change most often should be placed lower in the file. Also you can copy just requirements first to install libs, and then the whole project separately.